### DISCUSSION TITLE
Link instead of Copy/Paste

### DISCUSSION THREAD

**Discussion by theianjones** [johnlindquist/kit#300] [Ideas]
[This](https://github.com/hanbzu/link-instead/pull/1) seems like a perfect use case for script kit. Since it can detect everything that the @hanbzu is describing. 

I dont have much else to share other than I would love for this to exist.

**Comment by johnlindquist** [comment]
Sounds like OS-wide bookmarking.

1 and 2 are definitely possible for some/most apps.

3 is a pipe dream since context/sections are always changing and reading stuff like "cursor position" in apps just isn't possible. An alternative would be a screenshot of the state of the app that appears as you're browsing your bookmarks.

I'll put it on my list 📝

**Reply by hanbzu** [reply]
To be honest, I fully see that no 3 (sub-app navigation, even selections) would be only doable if apps supported it, especially those that support reviving sub-app context through `x-callback-url`. They would need to go further and post that URL somewhere accessible as the user navigates.

But what I'd love is something practical and doable that sets out the case for linking instead of copy-pasting.

So **1 (app) + 2 (filename) would already be very useful** (I have no idea how to do no 2).
